src/main/java/io/quarkus/search/app/quarkusio/QuarkusIO.java

```diff
@@ -214,9 +219,12 @@ private Guide createGuide(URI webUri, String version, String type, Map<String, O
         Guide guide = new Guide();
         guide.type = type;
         guide.title = toString(parsedGuide.get("title"));
-        guide.summary = toString(parsedGuide.get(summaryKey));
         guide.origin = toString(parsedGuide.get("origin"));
         guide.version = version;
+        String summary = toString(parsedGuide.get(summaryKey));
+        if (summary != null) {
+            guide.summary = markdownRenderer.render(markdownParser.parse(summary)).trim();
+        }
```

Comment:
Do you think it would make sense to render the title as well? Just to be future-proof?